base/internal/message.hpp

```diff
@@ -29,6 +30,7 @@ inline std::string DebugPrint(char t);
 
 template <typename U, typename V> inline std::string DebugPrint(std::pair<U, V> const & p);
 template <typename T> inline std::string DebugPrint(std::list<T> const & v);
+template <typename T> inline std::string DebugPrint(boost::circular_buffer<T> const & v);
```

Comment:
Please put this under all methods dealing with std::

Reply:
Done